supplements/critical-role-call-of-the-netherdeep/items.xml

```diff
@@ -0,0 +1,624 @@
+				<li>The jewel has 3 charges and regains all its expended charges daily at dawn. While holding the jewel, you can expend 1 charge from it to cast the <i>invisibility</i> spell.</li>
+			</ul>
+			<p class="indent"><strong><em>Awakened State.</em></strong> In this state, the jewel has received the blessing of Avandra the Change Bringer. Three delicate spires unfurl from the jewels center, like the buds of flowers opening in the spring. Three lapis lazuli stones rest like dewdrops on these spires.</p>
+			<p class="indent">The following benefits of the jewel improve:</p>
+			<ul>
+				<li>The bonus that the jewel confers to your AC increases to +2.</li>
+				<li>Its number of charges increases to 5.</li>
+			</ul>
+			<p class="indent">The jewel gains the following additional properties, which you can use while wearing or holding it:</p>
+			<ul>
+				<li>You can expend 1 of the jewels charges (no action required) to end one of the following conditions on yourself: grappled, paralyzed, or restrained.</li>
+				<li>When another creature you can see within 60 feel of you fails a saving throw, you can expend 1 of the jewel's charges as a reaction to enable that creature to reroll the saving throw, potentially turning a failure into a success. The creature must use the new roll.</li>
+			</ul>
+			<div class="sidebar">
+				<h5>VESTIGES OF DIVERGENCE</h5>
+				<p>When the great wars of the Calamity rolled across Exandria, heroes of divinity accepted legendary relics from their patrons and gods. At the same time, archmages wove dangerous arcane power into items of immense magical might. When the war ended, most of these magic items were buried with their wielders beneath ash and dust. Those items that remained accessible were passed down as heirlooms. These remnants of a lost era are known as the Vestiges of Divergence</p>
+				<p class="indent">Each Vestige of Divergence is a magic item that grows in power over time. When it is first encountered, the item is usually in its Dormant State. Either on its own or in response to the heroic deeds of its bearer, a Vestige of Divergence can enter its Awakened State. In time, it can reach its ultimate potential and enter its Exalted State, usually in response to a momentous event or some heroic achievement on the part of its bearer. The properties gained at each new state are cumulative with the properties of a previous state, unless otherwise noted.</p>
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility, reaction</set>
+			<set name="attunement">true</set>
+			<set name="charges">5</set>
+			<set name="cost" currency="gp">0</set>
```

Comment:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it